src/renderer/lines.rs

```diff
@@ -105,7 +105,7 @@ impl<'a> Lines<'a> {
     }
 }
 
-/// Create a rectangle between two cells.
+/// Create a rectangle spanning two cells.
```

Comment:
I don't think `spanning` two cells is the correct word here either. If the start cell is at index `0` and the end cell is at index `5`, the underline would go from `0` to the end of `5`. So it's not just from `0` to `1` or two other neighboring cells.

Maybe something like this is more understandable:

```rust
/// Create a vertical rectangle from the start of one, to the end of another cell.
```

I'm open to put even more information into this comment though. So there's no need to limit it to a short one-liner.

Reply:
updated